### PR TITLE
JDF-487: Remove -as- from quickstart artifactId and archive names and access URLs in README files

### PR DESCRIPTION
**Pull request by sgilda**

### CODE REVIEW

**Comment by sgilda** [comment]
I am testing the build now. I ran into failures with submodule quickstarts, so I have commented them out and am trying again.

**Comment by sgilda** [comment]
@rafabene : This builds successfully. I think it is ready for review.

File: ejb-multi-server/README.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ Also the client show how to invoke an EJB by using the scoped-context introduced
 
 5. If it is necessary to invoke the client with a different JBoss version the main class can be invoked by using the following command from the root directory of this quickstart. Replace $JBOSS_HOME with your current installation path. The output should be similar to the previous mvn executions.
 
-      java -cp $JBOSS_HOME/bin/client/jboss-client.jar:../../../ejb-multi-server/jboss-as-quickstart/ejb-multi-server/app-main/ejb/target/ejb-client.jar:../../../ejb-multi-server/jboss-as-quickstart/ejb-multi-server/app-two/ejb/target/ejb-client.jar:client/target/jboss-as-ejb-multi-server-client-7.1.2-SNAPSHOT.jar org.jboss.as.quickstarts.ejb.multi.server.Client
```
Comment (rafabene) [inline ejb-multi-server/README.md:194]: Good catch here!

File: shopping-cart/client/pom.xml [hunk 2]
```diff
@@ -21,11 +21,11 @@
 
     <parent>
         <groupId>org.jboss.quickstarts.eap</groupId>
-        <artifactId>jboss-as-shoppingcart-parent</artifactId>
```
Comment (rafabene) [inline shopping-cart/client/pom.xml:24]: Another good catch!!!

**Comment by rafabene** [comment]
It looks fine for me!

**Comment by sgilda** [comment]
Thanks @rafabene . I'll merge it.

**Pull request merged**